### DISCUSSION TITLE
Changes in repository shows another account

### DISCUSSION THREAD

**Discussion by majisoubhik01** [community/community#22149] [New to GitHub]
<p>Hello there,<br>
The issue that I am facing is that, I previously had an account signed in on my GitHub Desktop app but I recently opened another account for official purposes and I signed in with that on GitHub Desktop.<br>
Now, when I do some changes locally and then commit and push to origin, the updates are showing that they are made from my previous account and not from my original account.<br>
I tried uninstalling and reinstalling the GitHub Desktop app but the problem still persists.<br>
Can anybody tell me how I can change any setting so that it displays my account’s name on the commits when I do some changes locally?<br>
Thank you!</p>

**Comment by steveward** [comment, marked as answer]
<p>Welcome to the GitHub Community Forum <a class="mention" href="/mayhemongithub">@mayhemongithub</a>! It sounds like you still have the email address associated with your previous account set in your .gitconfig.</p>
<p>You can read about this issue here:</p>
<p><a href="https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user" class="onebox" target="_blank">https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user</a></p>
<p>You can check the email address that GItHub Desktop is currently using by going to the file menu and selecting <code>GitHub Desktop</code> &gt; <code>Options</code> &gt; <code>Git</code>. Make sure the email address matches an email address under your GitHub account. You can view your current email addresses here:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/settings/emails" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/721500/181069051-63f0f380-1905-44e2-9dc1-755ca5ddce9a.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://github.com/settings/emails" target="_blank">Build software better, together</a></h3>

<p>GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I hope that helps!</p>
